index.html

```diff
@@ -8,7 +8,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
     <title>LinuxChix India</title>
     <meta name="description" content="">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
```

Comment:
Hi, 

I completely respect your point of view. But I'm unable to understand how it won't restrict on the accessibility issues? 

Doing responsive design is one side of the coin and managing accessibility is completely different aspect. There are many devices, and we can't control where the site would be displayed. While we may arrange the size of text and images according to various view port sizes, different users might have different needs and might want to zoom in/ out.

I may be wrong here, and so would like to understand what is a good way to go. Tagging in some front-end devs. 

@slayslot @a2batic @shruti9520